deepchecks/checks/integrity/new_label.py

```diff
@@ -81,11 +81,12 @@ def add_condition_new_labels_not_greater_than(self, max_new: int = 0):
         def condition(result: Dict) -> ConditionResult:
             if result:
                 column_name = result['column_name']
-                num_new_labels = len(result['new_labels'])
+                new_labels = result['new_labels']
+                num_new_labels = len(new_labels)
                 if num_new_labels > max_new:
                     return ConditionResult(False,
-                                           f'Found more than {max_new} new labels in label column: '
-                                           f'{column_name}')
+                                           f'Found {num_new_labels} new labels in {column_name} column: '
```

Comment:
As long as the % of new labels is displayed in the check display, I think this is good. I don't think that the percent is needed here.

Reply:
I changed it from max number to actual number, should I still remove it?